tools/sz_repo_cli/test/ensure_cmd_sot_file_exists_test.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:sz_repo_cli/src/common/common.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Ensure source of truth yaml file exists', () async {
+    final projectRoot = await getProjectRootDirectory();
+
+    final repo = SharezoneRepo(projectRoot);
+
+    /// If the file gets moved and the path for
+    /// [repo.commandsSourceOfTruthYamlFile] not updated this will cause the CI
+    /// pipeline to fail.
```

Comment:
Is it correct to use there a documenation comment (`///`) instead of a normal comment (`//`)?

Reply:
I used it because I thought then the `[repo.commandsSourceOfTruthYamlFile]` would get automatically updated when the attribute name changes, but because its inline with the code it doesn't.   
I changed it to a normal comment.